### PR TITLE
Turn air-water-vv tests back on

### PR DESCRIPTION
**Pull request by zhang-alvin**
# Mandatory Checklist

Please ensure that the following criteria are met:

- [x] Title of pull request describes the changes/features
- [ ] Request at least 2 reviewers
- [x] If new files are being added, the files are no larger than 100kB. Post the file sizes.
- [x] Code coverage did not decrease. If this is a bug fix, a test should cover that bug fix. If a new feature is added, a test should be made to cover that feature.
- [x] New features have appropriate documentation strings (readable by sphinx)
- [x] Contributor has read and agreed with CONTRIBUTING.md and has added themselves to CONTRIBUTORS.md 

As a general rule of thumb, try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.

# Description

PR to turn AVV tests back on. Contingent on erdc/air-water-vv#233 being merged in. 

Also migrates Kappa2D to use xtensor structures.

### CODE REVIEW

**Comment by zhang-alvin** [comment]
All that remains then is to update the submodule to track the updated air-water-vv

**Comment by cekees** [comment]
OK, let's go ahead.

File: .travis.yml [hunk 1]
```diff
@@ -57,6 +57,9 @@ jobs:
     script:
       - export MPLBACKEND="AGG"
       - PATH=./linux/bin:$PATH py.test -n 1 --dist=loadfile --forked -v proteus/tests --ignore proteus/tests/POD -k 'not test_damBreak_solver_options' --cov=proteus
+      - git submodule update --init --recursive air-water-vv
+      - PATH=./linux/bin:$PATH py.test MPLBACKEND=Agg -n 1 --dist=loadfile --forked -v air-water-vv/Tests/1st_set
```
Comment (cekees) [inline .travis.yml:61]: I think the MPLBACKEND=Agg is confusing pytest.

Comment (zhang-alvin) [reply .travis.yml]: Yes I think that's the culprit as well. I just pushed the modification. Hopefully that's all it is.

**Pull request merged**